ert_storage/endpoints/records.py

```diff
@@ -227,7 +224,10 @@ async def post_ensemble_record_matrix(
         )
         content_type = "text/csv"
 
-    if prior_id is not None and record_class != "parameter":
+    ensemble = _get_and_assert_ensemble(db, ensemble_id, name, realization_index)
+    is_parameter = name in ensemble.parameter_names
+    is_response = name in ensemble.response_names
```

Comment:
Would it be totally stupid to validate only one of these are `True` at this point just to avoid some unwanted state later on? Is it impossible having a name which exists both in parameter and response?

Reply:
I'll add a check to the Ensemble